src/Farmer/Builders/Builders.Dns.fs

```diff
@@ -83,9 +102,13 @@ type DnsARecordBuilder() =
     [<CustomOperation "target_resource">]
     member _.RecordTargetResource(state:ARecordProperties, targetResource) = { state with TargetResource = Some targetResource }
 
+    /// Builds a record for an existing DNS zone.
+    [<CustomOperation "link_to_dns_zone">]
```

Comment:
Nit: Is there a convention for the name of this operation? I think usually the link_to_dns_zone would imply that its linking to a managed resource and it should take in a IBuilder while link_to_unmanaged_dns_zone would imply that its linking to an externally deployed dns zone.